machine/v1/types_powervsprovider.go

```diff
@@ -190,3 +204,9 @@ type PowerVSSecretReference struct {
 	// +optional
 	Name string `json:"name,omitempty"`
 }
+
+// LoadBalancerReference is a reference to a load balancer on IBM Cloud virtual private cloud.
+type LoadBalancerReference struct {
+	Name string                 `json:"name"`
+	Type IBMVPCLoadBalancerType `json:"type"`
+}
```

Comment:
These are both required fields? Can you please add some godoc to them, following the guidance for [writing user readable documentation](https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#write-user-readable-documentation-in-godoc)

Reply:
Both are required fields and added some documentation. Please take a look.